components/Chat/Chat.vue

```diff
@@ -254,11 +297,14 @@
               page: this.page,
             };
 
-            await this.connectChat( tokenTroll );
+            await this.disconnectChat();
+            this.userToken = tokenTroll;
+            await this.connectChat();
           }
         }
       },
 
+      // TODO: why are these here?
```

Comment:
to detect when user signs out in another tab.

Reply:
'why here' as in 'why this file?' lol
I get *what* it does